swingset/src/main/java/com/nqadmin/swingset/formatting/SSMaskFormatterFactory.java

```diff
@@ -0,0 +1,418 @@
+/* *****************************************************************************
```

Comment:
New method so will probably try to digest in Eclipse rather than in GitHub site.

Reply:
FYI, SSMaskFormatterFactory had no changes from the first preview you saw.